data/editor.config

```diff
@@ -1201,6 +1201,9 @@
         <include field="operator" />
         <include field="internet" />
       </type>
+      <type id="highway-ladder">
+        <include field="height" />
```

Comment:
this is an interesting idea (and not technically wrong), but isn't documented on the wiki & there are only 12 `highway=ladder` with `height` tagged worldwide. It'd be better to remove this i think